src/pymovements/datasets/etdd70.yaml

```diff
@@ -0,0 +1,57 @@
+name: ETDD70
+
+has_files:
+  gaze: true
+  precomputed_events: true
+  precomputed_reading_measures: false
+
+mirrors:
+  gaze:
+      - https://zenodo.org/api/records/13332134/files-archive
+  precomputed_events:
+      - https://zenodo.org/api/records/13332134/files-archive
+
+resources:
+  gaze:
+    - resource: ''
```

Comment:
I don't get how that should work without specifying an URL? have you tested this?

Reply:
ah, probably because mirror is already the full path?

Reply:
does this mean the dataset is downloaded twice?

Reply:
yes, I'll change to mirror only. its the same issue as in the other PR

Reply:
#1136